docs/proposals/TEMPLATE.md

```diff
@@ -34,6 +34,6 @@ it from another react component? This should consist primarily of code blocks}_
 
 _{Provide a table in the following format of the component's public API}_
 
-| name | type | default | description |
-| ---- | ---- | ------- | ----------- |
-| ...  | ...  | ...     | ...         |
+| name | type | required | default | description |
```

Comment:
Consider `required?` (with the question mark) to hint at the boolean nature/expectation....